test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-ui-state.json

```diff
@@ -202,12 +202,7 @@
     "isEvmSelected": "boolean",
     "networksWithTransactionActivity": "object",
     "selectedNetworkClientId": "string",
-    "networksMetadata": {
-      "networkConfigurationId": {
```

Comment:
Is it intentional that we're hiding this information in Sentry snapshots now?

Reply:
@Gudahtt I see now why this was removed because of this ci failure https://github.com/MetaMask/metamask-extension/actions/runs/19773997500/job/56664038305?pr=37966

Reply:
See https://github.com/MetaMask/metamask-extension/pull/37966#discussion_r2577702136 for context.

@cryptodev-2s `networksMetadata` still shows up in state so I figured that we still wanted to represent it in some form in these snapshots. What do you thikn?